packages/osquery_manager/kibana/osquery_saved_query/osquery_manager-d8a1b2c3-d4e5-11ef-a6b7-12c3d4e5f678.json

```
@@ -0,0 +1,87 @@
+{
+    "attributes": {
+        "created_at": "2025-11-29T00:00:00.000Z",
+        "created_by": "elastic",
+        "description": "Retrieve physical disk hardware information on Windows systems including manufacturer, model, serial number, size, and interface type. Useful for asset inventory and hardware verification.",
+        "ecs_mapping": [
+            {
+                "key": "event.category",
+                "value": {
+                    "value": ["host"]
+                }
+            },
+            {
+                "key": "event.type",
+                "value": {
+                    "value": ["info"]
+                }
+            },
+            {
+                "key": "event.module",
+                "value": {
+                    "value": "osquery"
+                }
+            },
+            {
+                "key": "event.dataset",
+                "value": {
+                    "value": "osquery.disk_info"
+                }
+            },
+            {
+                "key": "host.os.type",
+                "value": {
+                    "value": "windows"
+                }
+            },
+            {
+                "key": "host.disk.device.name",
```

This field doesn't exist and the data seems to the be local path the drive is mounted? Maybe `file.path`?

```suggestion
                "key": "file.path",
```